src/main/java/seedu/address/model/group/Group.java

```diff
@@ -14,8 +14,8 @@
 public class Group {
 
     public static final String MESSAGE_CONSTRAINTS = "Group is written as g/GROUP_NAME "
-            + "(less than 16 characters incl spaces between words), please ensure there are no empty group names too!";
-    public static final String VALIDATION_REGEX = "^.{1,15}$";
+            + "(100 characters or less, incl spaces between words), please ensure there are no empty group names too!";
```

Comment:
Maybe can just make this an error message since a typical user likely won't hit the 100 char cap